src/Components/test/E2ETest/Infrastructure/SeleniumStandaloneServer.cs

```diff
@@ -40,15 +43,30 @@ private SeleniumStandaloneServer()
             var port = FindAvailablePort();
             Uri = new UriBuilder("http", "localhost", port, "/wd/hub").Uri;
 
-            var process = Process.Start(new ProcessStartInfo
+            var psi = new ProcessStartInfo
             {
                 FileName = "npm",
                 Arguments = $"run selenium-standalone start -- -- -port {port}",
-                UseShellExecute = true,
-            });
+                RedirectStandardOutput = true,
+                RedirectStandardError = true,
+            };
+
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                psi.FileName = "cmd";
+                psi.Arguments = $"/c npm {psi.Arguments}";
+            }
 
-            PollUntilProcessStarted();
+            var process = Process.Start(psi);
```

Comment:
Didn't we have something to do this in a shared source/helper?

Reply:
Maybe, but all of them do things ever so slightly differently (the timeouts are different and I test the Razor one because it's been thru several iterations of bug fixes)